Automattic/jetpack
# Welcome dialogue for 5.8

This will add a welcome dialogue to the 5.8 release. 

To Test: 
- Manually change the `JETPACK__VERSION` constant in jetpack.php to 5.8
- Load the Jetpack admin dashboard
- See the dialogue
- Should not be there on reload.  

~(in progress until we have the copy)~
(in progress until we have updated images)
